OISF/suricata-verify
# detect-flowbits/v5: add tests

Adds test for FlowBitsTestSig6-11

Changes:
- Check that toggle keyword works
- remove unnecessary rules

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/4911

Previous PR: #703

Corresponding Suricata PR: https://github.com/OISF/suricata/pull/6899
